analysis/analysis.py

```diff
@@ -51,11 +51,103 @@ def run_analysis():
     """
     csv_path = Path('data', 'rereading_data_2019-09-13.csv')
     student_data = load_data_csv(csv_path)
-
+    response_groups_freq_dicts = get_response_groups_frequencies(student_data)
+    show_response_groups(response_groups_freq_dicts)
     total_view_time = compute_total_view_time(student_data)
     print(f'The total view time of all students was {total_view_time}.')
 
 
+def show_response_groups(response_groups_freq_dicts):
+    """
+    Given response_group dictionary, prints the dicts in readable format
+
+    :param response_groups_freq_dicts, keys are four dicts (one for each response group)
+    mapping words to frequencies within that response group
+    :return None
+    """
+    # print(f'Word frequencies for {group_name}: {response_groups_freq_dicts[group_name]}\n')
+    print(f'Word frequencies for Single view responses to ad context:',
+          response_groups_freq_dicts[0])
+    print(f'Word frequencies for Single view responses to short story context:',
+          response_groups_freq_dicts[1])
+    print(f'Word frequencies for Multiple view responses to ad context:',
+          response_groups_freq_dicts[2])
+    print(f'Word frequencies for Multiple view responses to short story context:',
+          response_groups_freq_dicts[3])
+
+
+def get_response_groups_frequencies(student_data: list):
+    """"
+    Given student_data,
```

Comment:
This docstring is now outdated -- plz update to reflect your refactor into a list of dicts

Reply:
(I mean that the return is a list of dicts, not the inputs!)